x/tokenomics/keeper/token_logic_modules.go

```diff
@@ -815,16 +832,16 @@ func calculateGlobalPerClaimMintInflationFromSettlementAmount(settlementCoin sdk
 	return mintAmtCoin, *newMintAmtFloat
 }
 
-// stakeToMaxSettlementAmount calculates the max amount of uPOKT to that the supplier
+// stakeShareToMaxSettlementAmount calculates the max amount of uPOKT to that the supplier
 // can claim based on the stake share and the global inflation allocation percentage.
```

Comment:
I still don't understand what `stakeShare` is?

- Is it an amount? If so, why is it called a "share"
- Is it a percentage/fraction? If so, a fraction of "what"

Only blocking this PR because I'm unclear on this topic, so let's find a better name.

Reply:
It's the supplier's application stake "share" to max settlement amount.

I renamed it appropriately.